src/Runner/Runner.php

```diff
@@ -95,7 +90,7 @@ public function run()
 			$handler->begin();
```

Comment:
More precisely, the most perfect scenario would be if I could get an array of all `Test`s so that the output handler could process them (similarly to what I did in #335). It would make sense to me if `findTests` created and actually returned found `Test`s, and the `while` loop iterated over them instead of `Job`s. But that would mean that `Test`s would somehow have to be aware of their respective `Job`s, which is something you [didn't quite like](https://github.com/nette/tester/pull/334/files#r78320336)